bundles/org.dataflowanalysis.analysis/src/org/dataflowanalysis/analysis/core/AbstractPartialFlowGraph.java

```diff
@@ -1,11 +1,6 @@
 package org.dataflowanalysis.analysis.core;
 
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
```

Comment:
I honestly am not a big fan of .* imports. Is there a specific reason?

Reply:
Fixed in 7c0133255b70cf6033cef879fdb862a6d8efd8c9